manifests/config.pp

```diff
@@ -31,8 +31,14 @@
   }
 
   file { $::foreman::init_config:
```

Comment:
I was wondering if we should keep this in or just leave the file. If we modify the loading in RPM packaging to stop reading this file, we can drop this.